### PR TITLE
ADEN-6972 Add geo with sampling

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/context/adContext.js [hunk 1]
```diff
@@ -215,6 +216,10 @@ define('ext.wikia.adEngine.adContext', [
 		context.opts.disableSra = geo.isProperGeo(instantGlobals.wgAdDriverDisableSraCountries);
 		context.opts.isBLBMegaEnabled = geo.isProperGeo(instantGlobals.wgAdDriverBottomLeaderBoardMegaCountries);
 
+		context.opts.labradorTestGroup =
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/context/adContext.js]: Where is it used?

Comment (v3rth) [reply extensions/wikia/AdEngine/js/context/adContext.js]: It isn't, it is defined only for testing purposes

File: extensions/wikia/AssetsManager/config.php [hunk 2]
```diff
@@ -219,6 +219,7 @@
 		'//extensions/wikia/AdEngine/js/utils/eventDispatcher.js',
 		'//extensions/wikia/AdEngine/js/utils/sampler.js',
 		'//extensions/wikia/AdEngine/js/utils/timeBuckets.js',
+		'//extensions/wikia/AdEngine/js/build/bridge.js',
```
Comment (nandy-andy) [inline extensions/wikia/AssetsManager/config.php]: Yet another reason to move to AE3 ASAP :japanese_ogre:

Comment (fraszczakszymon) [inline extensions/wikia/AssetsManager/config.php]: 😱 is it really really needed here? It will affect all bidders performance (?)

**Comment by v3rth** [comment]
Tests will pass after merge dev https://github.com/Wikia/app/commit/9c0c5d230755051c1ce76547d81609b5b1b389c8 . I can't do this right now, because it will break versions of dependencies

**Pull request merged**